tests/acceptance/install_test.sh

```diff
@@ -65,7 +65,7 @@ function test_install_downloads_the_non_stable_beta_version() {
     "$output"
   assert_file_exists "$installed_bashunit"
   assert_matches\
-    "$(printf "\(non-stable\) beta after ([0-9]+\.[0-9]+\.[0-9]+) \[2023-11-13\]")"\
+    "$(printf "\(non-stable\) beta after ([0-9]+\.[0-9]+\.[0-9]+) \[2023-11-13\] 🐍 \#[0-9aZ]+")"\
```

Comment:
```suggestion
    "$(printf "\(non-stable\) beta after ([0-9]+\.[0-9]+\.[0-9]+) \[2023-11-13\] 🐍 \#[0-9aZ]{8}")"\
```

If I try the `{8}`, the pipeline fails, I don't know if there is a way to define it.